openedx/tests/completion_integration/test_views.py

```diff
@@ -31,7 +31,7 @@ def setUp(self):
         Create the test data.
         """
         super(CompletionBatchTestCase, self).setUp()
-        self.url = reverse('completion_api:v1:completion-batch')
+        self.url = reverse('completion:v1:completion-batch')
 
```

Comment:
The separate-repo completion api uses the "completion" namespace, where it had been "completion_api" when it was here.